.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,3 +3,11 @@
 ### Changes proposed in this pull request
 
 ### Guidance to review
+
+### Checklist
+
+- [ ] Make sure all information from the Trello card is in here
+- [ ] Attach to Trello card
+- [ ] Rebased master
```

Comment:
Doesn't GitHub take care of this for us?

Reply:
Only when merging. I threw it in mostly to remind people to rebase off the latest master when doing a PR, in case there are conflicts/breaking changes. Happy to remove it though.